File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.java
import java.util.*;
class Solution {
    public int[] solution(int[] arr, int divisor) {
        int count = 0;
        for(int i=0; i<arr.length; i++){
            if(arr[i]%divisor == 0){
                count++;
            }
        }
        if(count == 0){
            int[] answer = {-1};
            return answer;
        }
        int[] answer = new int[count];
        int n = 0;
        for(int i=0; i<arr.length; i++){
            if(arr[i]%divisor == 0){
                answer[n] = arr[i];
                n++;
            }
        }
        Arrays.sort(answer);
        return answer;
    }
}

File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
    let count = 0;
    for(let i=0; i<arr.length; i++){
            if(arr[i]%divisor == 0){
                count++;
            }
        }
        if(count == 0){
            var answer = [-1];
            return answer;
        }
        var answer = new Array(count);
        let n = 0;
        for(let i=0; i<arr.length; i++){
            if(arr[i]%divisor == 0){
                answer[n] = arr[i];
                n++;
            }
        }
        answer.sort((a,b) => a-b);
    return answer;
}